<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumesIOOptionsWidget</class>
 <widget class="QWidget" name="qSlicerVolumesIOOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="CenteredCheckBox">
     <property name="text">
      <string>Centered</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="OrientationCheckBox">
     <property name="text">
      <string>Ignore Orientation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="LabelMapCheckBox">
     <property name="text">
      <string>LabelMap</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="SingleFileCheckBox">
     <property name="text">
      <string>Single File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
